README.md

```diff
@@ -38,6 +38,7 @@ This VS Code extension provides support for creating and editing XML documents,
   * XSD based hover
   * XSD based code completion
   * XSL support
+  * [XInclude validation](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features/XIncludeFeatures.md#Validation)
```

Comment:
I will rename to `XInclude support`, because I think we will support documentLink feature, not only validation